### ISSUE TITLE
[Bug]: fimsframe R test failed to run on codespace

### ISSUE THREAD

**Issue by Bai-Li-NOAA** [NOAA-FIMS/FIMS#360]
### Describe the bug

When running devtools::check() or devtools::test() on FIMS codespace (Ubuntu 22.04.2 LTS), the fimsframe test failed to run. 
```r
══ Failed tests ════════════════════════════════════════════════════════════════
── Failure ('test-fimsframe.R:24:3'): Can create the S4 FIMSFrame classes ──────
`save_png(plot(fims_frame))` produced warnings.

[ FAIL 1 | WARN 9 | SKIP 0 | PASS 2666 ]
Error: Test failures
Execution halted
```

### To Reproduce

Go to FIMS codespace and then run devtools::check() on R interactive terminal.

### Expected behavior

All R tests should pass on Ubuntu. See [log](https://github.com/NOAA-FIMS/FIMS/actions/runs/5060110228/jobs/9082568320) from GHA R CMD Check. 

### Which OS are you seeing the problem on?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by k-doering-NOAA** [comment]
I noticed this before, too, when I was using the codespace.

**Comment by Bai-Li-NOAA** [comment]
I was able to print out the warning message from the fimsframe test:
```r
Warning (test-fimsframe.R:24:3): Can create the S4 FIMSFrame classes
unable to load shared object '/usr/local/lib/R/modules//R_X11.so':
  libXt.so.6: cannot open shared object file: No such file or directory
```
The **solution** to this issue is to run the following code in codespace bash
```bash
sudo apt-get update -y
sudo apt-get install -y libxtst6
sudo apt-get install -y libxt6
```
Then devtools::test() runs successfully on codespace.

@ChristineStawitz-NOAA and @k-doering-NOAA, I am able to fix the issue on my codespace session by manually installing libxtst6 and libxt6. Could you add libxtst6 and libxt6 to FIMS devcontainer and see if it works in your codespace session?

**Comment by ChristineStawitz-NOAA** [comment]
Thanks @Bai-Li-NOAA - I added this to the codespace devcontainer on branch 360-bug-fimsframe... but there is an unrelated issue with the devcontainer where it looks like it can't find or start an R terminal. I tried adding `r.rterm.linux: usr/bin/radian` to the .devcontainer to no avail. It looks like this issue happens in codespaces on all branches. I'll try to look at it again tomorrow!
